### DISCUSSION TITLE
Maintenance for all endpoints

### DISCUSSION THREAD

**Discussion by jpleasants** [upptime/upptime#828] [Q&A]
We host websites on individual servers and have created different repos per server with all respective sites listed on each status site. As such, if there's maintenance on the server that is expected to bring all sites down, in the maintenance issue, is it possible to use a wildcard for expectedDown or if it's not defined will all be implied?

``

``

**Comment by AnandChowdhary** [comment]
Unfortunately this is not supported as we parse the comma-separated string: https://github.com/upptime/uptime-monitor/blob/854c36315604b947af62e5a2af87e2549785a177/src/update.ts#L62
